examples/servients/coap-servient/package.json

```diff
@@ -20,7 +20,7 @@
     "devDependencies": {
         "wot-typescript-definitions": "^0.8.0-SNAPSHOT.18",
         "@types/node": "16.4.13",
-        "typescript": "4.1.3"
+        "typescript": "4.4.3"
```

Comment:
4.4.3 is in fact now need

Reply:
Good, next time please state this in the PR description. It's our own limitation cause we don't provide a PR template, but I think that changes to dependences should be always justified. See #559.